Content.Server/_NF/ShuttleRecords/ShuttleRecordsSystem.cs

```diff
@@ -97,21 +98,21 @@ public bool TrySetSaleTime(NetEntity uid)
         }
 
         StringBuilder builder = new();
-        Dictionary<String, RecordSummary> shipTypes = new(); // committing crimes against structs here
+        Dictionary<string, RecordSummary> shipTypes = new(); // committing crimes against structs here
         var totalShips = 0;
         var totalAbandoned = 0;
         List<TimeSpan> totalLifetimes = new();
 
         // sort through the records and use originalname to categorise ships
```

Comment:
```suggestion
        // sort through the records and use VesselPrototypeId to categorise ships
```
may as well be consistent about the comments